### PR TITLE
Corrected grammar & <required> element

### PR DESCRIPTION
**Pull request by gfischershaw**
Made some grammar corrections and removed a couple instances wherein the text slipped into second-person. It also appears that the "required" attribute is now an element, at least based on what I just downloaded.

### CODE REVIEW

File: docs/sources/howtos/bindings.md [hunk 1]
```diff
@@ -60,11 +60,12 @@ Eclipse SmartHome allows you to define your *ThingTypes* in a declarative way th
         <channel id="temperature" typeId="temperature" />
     </channels>
     <config-description>
-        <parameter name="location" type="text" required="true">
```
Comment (dnobel) [inline docs/sources/howtos/bindings.md]: We changed "required" to be an attribute. The xml element remained just for backward compatibility. But some bindings needs to be adapted.

**Comment by kaikreuzer** [comment]
Thanks @gfischershaw and sorry for taking so long - I had to clarify, whether I am allowed to merge the PR. The answer is that I would need you to sign off your commit (see http://stackoverflow.com/questions/1962094/what-is-the-sign-off-feature-in-git-for) in order to be able to pull it in. May I ask you to thus amend your existing commit and also take @dnobel's comment into account? Thanks!

**Comment by kaikreuzer** [comment]
I have included the suggestions in https://github.com/eclipse/smarthome/pull/226

**Comment by gfischershaw** [comment]
I see no way to do this in the GUI. If not a simple process, then just delete, as this is trivial compared to the enormous lack of documentation overall.

**Comment by kaikreuzer** [comment]
No worries, as mentioned in https://github.com/eclipse/smarthome/pull/194#issuecomment-83185190, I have already updated the docs with your changes - thanks again!

**Pull request closed without merging**